code/lightna/lightna-engine/js/lib/HttpClient.js

```diff
@@ -68,33 +84,10 @@ class HttpClient {
         return responseJson;
     }
 
-    get(url, options = {}) {
-        return this.fetch(url, {
-            ...options,
-            method: 'GET',
-        });
-    }
-
-    post(url, data, options = {}) {
-        if (!data.form_key) {
-            data.form_key = FormKey.get();
-        }
```

Comment:
form key actually is also magento dependency, we magento need also to make it by some extend. could you make a ticket for it?